### ISSUE TITLE
distinguish Director of Board type

### ISSUE THREAD

**Issue by dericed** [amiaopensource/amia-org#11]
Within the section for the AMIA Board, I think the directors could be subdivided into:
- `Elected Directors`
- `Designated Directors`

AFAIK, (@retokromer?) there are currently no `Designated Directors` but the board has the power to appoint them up to a ratio of 2:1 Elected to Designated.

**Comment by retokromer** [comment]
Yes, there is currently a designated one: Randal Luckow. (We needed a new secretary, because Tyler Leshney had to step down for professional reasons.)

**Comment by retokromer** [comment]
Update: This has been the case until 2016-11-12.
